resources/server/messages/messages.service.ts

```diff
@@ -120,17 +120,18 @@ class _MessagesService {
 
       for (const participantId of userParticipants) {
         // we don't broadcast to the source of the event.
-        if (participantId !== _identifier) {
-          const participantPlayer = PlayerService.getPlayerFromIdentifier(participantId);
-          if (!participantPlayer) {
```

Comment:
If you remove this return, whenever you send a message to an offline player you will get the catch statement, which is not intended in this case because its not an error, it's just the broadcast and the refresh that won't be triggered for the offline receiver.

Reply:
Well, as talked offline, not only don't remove the return, but also changing it to a "continue" should do the trick.